lib/src/components/board/board.dart

```diff
@@ -58,6 +70,12 @@ abstract class Board extends Resource {
   /// Return the current value of the interrupt which is based on the type of Interrupt.
   Future<int> digitalInterruptValue(String digitalInterruptName, {Map<String, dynamic>? extra});
 
+  // Stream digital interrupts ticks.
+  Future<ResponseStream<StreamTicksResponse>> streamTicks(List<String> interrupts, {Map<String, dynamic>? extra});
```

Comment:
Can we make the return type a `Stream` of `Tick`s please. Take a look at how we do it for [`tailLogs`](https://github.com/viamrobotics/viam-flutter-sdk/blob/337e9d2767eec68c022bca94a6f3c852b94b822b/lib/src/app/app.dart#L95) to get an idea of how to convert a `ResponseStream` to a native `Stream`